libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/CopilotOptions.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapbox.navigation.base.internal
+
+/**
+ * Defines options for Copilot.
+ *
+ * @param shouldSendHistoryOnlyWithFeedback true if Copilot History files should be sent only when they include Feedback events. Default is false
+ */
+class CopilotOptions private constructor(
```

Comment:
You should update current.txt.

Reply:
Nah, it's under the `internal` package and hence not publicly available (as discussed).

Reply:
Oh, right.